### PR TITLE
added check to proceed with installation only if nodejs and npm are not installed

### PR DESCRIPTION
**Pull request by sanathusk**
added check to proceed with installation only if nodejs and npm are not installed

### CODE REVIEW

**Review by NeoTheThird — DISMISSED** [review]
lgtm, thanks

File: setup-dev.sh [hunk 1]
```diff
@@ -3,6 +3,26 @@
...
+    isNodeInstalled=true
+fi
+if command -v npm &> /dev/null
+then
+    isNpmInstalled=true
+    echo "npm is already installed on the machine."
+fi
+if [ "$isNodeInstalled" = true ] && [ "$isNpmInstalled" = true ]
+then
+    echo "both node and npm is already installed on the machine.Nothing more to install."
```
Comment (amartinz) [inline setup-dev.sh:22]:
```suggestion
    echo "Both node and npm are already installed. Nothing more to install."
```

File: setup-dev.sh [hunk 2]
```diff
@@ -3,6 +3,26 @@
 distro=$(lsb_release -si)
 version=$(lsb_release -sr)
 
+isNodeInstalled=false
+isNpmInstalled=false
+
+echo "The script will try to install nodejs and npm on the machine."
+if command -v node &> /dev/null
+then
+    echo "nodejs is already installed on the machine"
```
Comment (amartinz) [inline setup-dev.sh:12]:
```suggestion
    echo "nodejs is already installed."
```

File: setup-dev.sh [hunk 3]
```diff
@@ -3,6 +3,26 @@
...
+    isNodeInstalled=true
+fi
+if command -v npm &> /dev/null
+then
+    isNpmInstalled=true
+    echo "npm is already installed on the machine."
```
Comment (amartinz) [inline setup-dev.sh:18]:
```suggestion
    echo "npm is already installed."
```

**Comment by amartinz** [comment]
@maciek134 brought up that we may nuke this script altogether.

Thoughts on this, @NeoTheThird? :)

**Comment by amartinz** [comment]
`setup-dev.sh` is gone since https://github.com/ubports/ubports-installer/commit/a8fcd05f87b3705dcbe46cb5b0abc4a09ba0ed11

Closing this PR, thanks for it though! :)

**Pull request closed without merging**